src/model/openthaigpt_pretraining_model/models/constants.py

```diff
@@ -28,8 +28,10 @@
 LORA_CONFIG = {
     "llama_hf": LoraConfig,
     "gptj": LoraConfig,
+    "falcon": LoraConfig,
```

Comment:
I think you should use `LoraConfig` when you use lora and raise Value Error if model does not support Lora.

Reply:
this you mean just put all to loraconfig if it not support let Lora raise Value error?